scripts/setup-macos.sh

```diff
@@ -42,7 +42,7 @@ function run_and_time {
 function prompt {
   (
     while true; do
-      local input="${PROMPT_ALWAYS_RESPOND}"
+      local input="${PROMPT_ALWAYS_RESPOND:-}"
```

Comment:
what does this do?

Reply:
fix undefined variable error when this file was used by presto_cpp at https://github.com/facebookexternal/presto_cpp/blob/c96a88c89489574dba8b791c50a24b4a52a559ae/scripts/setup-macos.sh#L17, and the file has a -u flag. The error will cause rebuilding dependencies every time